### ISSUE TITLE
'renderDefaultValues' does not exist in type 'Options' 

### ISSUE THREAD

**Issue by JamesIves** [break-stuff/wc-storybook-helpers#71]
I just recently converted my Storybook instance to TypeScript, and I'm noticing type errors here when using `2.0.1`. I would expect what I have below to not produce an error as this option was introduced in `2.0`.

<img width="1126" alt="Screenshot 2024-05-06 at 8 56 35 AM" src="https://github.com/break-stuff/wc-storybook-helpers/assets/10888441/66474e13-7c06-4a37-baba-751baea70e64">

Looking through the installed `node_modules` folder for 2.0.1, I see the following present in `storybook.d.ts`:

```typescript
export interface Options {
  /** hides the `arg ref` label on each control */
  hideArgRef?: boolean;
  /** sets the custom type reference in the Custom Elements Manifest */
  typeRef?: string;
  /** Adds a <script> tag where a `component` variable will reference the story's component */
  setComponentVariable?: boolean;
  /** doesn't render attributes when their value is equal to the default value of that attribute */
  renderDefaultAttributeValues?: boolean;
}
```

Based on what is in the branch main, `renderDefaultAttributeValues` no longer exists and was replaced with `renderDefaultValues`, which makes me wonder if there was an issue with the initial publishing of 2.0.0 that is causing the types for the new option to not be included in the bundle. I tried downgrading to `2.0.0` and I see the same problem.

**Comment by break-stuff** [comment]
You're right. The type is different in the package for some reason. I redeploy to see if I can get it fixed.

**Comment by break-stuff** [comment]
This should now be fixed in release 2.0.2

**Comment by JamesIves** [comment]
Looks good! Thanks
